### PR TITLE
Feat/components

### PR DESCRIPTION
**Pull request by goyadev**
added in components skeletons for header, nav, footer and main. A starting point to flesh out the shape before adding the pretty

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,9 @@ yarn-error.log*
 pnpm-debug.log*
 lerna-debug.log*
 
+
+package-lock.json
+
```
Comment (suanTech) [inline .gitignore]: Please remove this from .gitignore, we want to commit package-lock.json now so that we can npm ci the latest packages.

**Review by suanTech — CHANGES_REQUESTED** [review]
Please resolve commented file.

File: .gitignore (hunk 1 above)
Comment (goyadev) [reply .gitignore]: removed

**Review by suanTech — APPROVED** [review]
Lgtm!!! 👍

**Pull request merged**